src/Uno.UI.Runtime.Skia.X11/X11DisplayInformationExtension.cs

```diff
@@ -68,13 +68,68 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
+// COPYRIGHT 1987, 1991
+// DIGITAL EQUIPMENT CORPORATION
+// MAYNARD, MASSACHUSETTS
+// MASSACHUSETTS INSTITUTE OF TECHNOLOGY
+// CAMBRIDGE, MASSACHUSETTS
+// ALL RIGHTS RESERVED.
+//
+// THE INFORMATION IN THIS SOFTWARE IS SUBJECT TO CHANGE WITHOUT NOTICE AND
+// SHOULD NOT BE CONSTRUED AS A COMMITMENT BY DIGITAL EQUIPMENT CORPORATION.
+// DIGITAL MAKES NO REPRESENTATIONS ABOUT THE SUITABILITY OF THIS SOFTWARE FOR
+// ANY PURPOSE.  IT IS SUPPLIED "AS IS" WITHOUT EXPRESS OR IMPLIED WARRANTY.
+//
+// IF THE SOFTWARE IS MODIFIED IN A MANNER CREATING DERIVATIVE COPYRIGHT RIGHTS,
+// APPROPRIATE LEGENDS MAY BE PLACED ON THE DERIVATIVE WORK IN ADDITION TO THAT
+// SET FORTH ABOVE.
+//
+// Permission to use, copy, modify, and distribute this software and its
+// documentation for any purpose and without fee is hereby granted, provided
+// that the above copyright notice appear in all copies and that both that
+// copyright notice and this permission notice appear in supporting
+// documentation, and that the name of Digital Equipment Corporation not be
+// used in advertising or publicity pertaining to distribution of the software
+// without specific, written prior permission.
+//
+// ----------------------------------------------------------------
+//
+// Copyright 1991, Digital Equipment Corporation.
+// Copyright 1991, 1994, 1998  The Open Group
```

Comment:
@jeromelaban Should we have any special handling for such external copyrights?

Reply:
This is preserving the intent of the original copyright.